src/Shared/Helpers/ArchiveHelper.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+
+namespace Microsoft.CST.OpenSource.Helpers;
+
+using RecursiveExtractor;
+using System;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+public static class ArchiveHelper
```

Comment:
Some basic unit tests for this new class would be good.

Reply:
Done.